packages/web/src/components/molecules/EntityGrid/EntityGridItemPure.tsx

```diff
@@ -108,13 +112,15 @@ const EntityGridItemPure = forwardRef<HTMLDivElement, EntityGridItemPureProps>((
           </ItemColumn>
           <ExecutionTimeItemColumn>
             <EntityGridItemExecutionTime time={latestExecution?.startTime} />
-            {isRunning ? (
-              <DotsDropdown
-                placement="bottomRight"
-                disabled={outOfSync}
-                items={[{key: 1, label: <span onClick={abort}>Abort all executions</span>}]}
-              />
-            ) : null}
+            <DropdownComponent
+              name={item.name}
+              namespace={item.namespace}
+              outOfSync={outOfSync}
+              type={item.type}
+              useGetMetricsQuery={
+                entityLabel === 'test' ? useGetTestExecutionMetricsQuery : useGetTestSuiteExecutionMetricsQuery
+              }
```

Comment:
Not needed, as the `DropdownComponent` may (and it does) have already the expected queries used directly
```suggestion
```

Reply:
Nice catch 👍🏻